autofs-5.0.7 - document allowed map sources in auto.master

From: Ian Kent <raven@themaw.net>

Direct map may have multiple map sources but indirect maps may have
only one map source.
---
 CHANGELOG            |    1 +
 man/auto.master.5.in |    7 +++++--
 2 files changed, 6 insertions(+), 2 deletions(-)

--- autofs-5.0.5.orig/CHANGELOG
+++ autofs-5.0.5/CHANGELOG
@@ -183,6 +183,7 @@
 - change walk_tree() to take ap.
 - add negative cache lookup to hesiod lookup.
 - fix use get_proximity() without libtirpc.
+- document allowed map sources in auto.master.
 
 03/09/2009 autofs-5.0.5
 -----------------------
--- autofs-5.0.5.orig/man/auto.master.5.in
+++ autofs-5.0.5/man/auto.master.5.in
@@ -40,7 +40,8 @@ For direct maps the mount point is alway
 .P
 /-
 .P
-and the key used within the direct map is the full path to the mount point.
+and the key used within the direct map is the full path to the mount point. The direct map
+may have multiple entries in the master map.
 .P
 For indirect maps access is by using the path scheme:
 .P
@@ -60,7 +61,9 @@ present in the master map by including a
 and 
 .BR automount (8)
 will process the map according to the specification described below for
-map entries.
+map entries. Indirect map entries must be unique in the master map so
+second and subsequent entries for an indirect mount point are ignored by
+.BR automount (8).
 .SH "FORMAT"
 Master map entries have three fields separated by an arbitrary number
 of spaces or tabs. Lines beginning with # are comments. The first field
